StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1618GenericTypeParametersMustBeDocumented.cs

```diff
@@ -18,6 +18,7 @@
     /// for one or more of its generic type parameters.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1622, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <typeparam name="S"></typeparam>
```